artifactory/utils/dependenciesutils.go

```diff
@@ -244,9 +248,11 @@ func createHttpClient(artDetails *config.ServerDetails) (rtHttpClient *jfroghttp
 	return
 }
 
-func getAnalyzerManagerRemoteDetails(downloadPath string) (server *config.ServerDetails, remoteRepo string, err error) {
-	server, remoteRepo, err = getRemoteDetailsFromEnv(downloadPath)
+func getAnalyzerManagerRemoteDetails(downloadPath string) (server *config.ServerDetails, fullRemotePath string, err error) {
+	var remoteRepo string
+	server, remoteRepo, err = getRemoteDetails(coreutils.ReleasesRemoteEnv)
 	if remoteRepo != "" || err != nil {
+		fullRemotePath = path.Join(remoteRepo, "artifactory", downloadPath)
```

Comment:
This line should not run if err is not nil.

Reply:
done